rllib/algorithms/appo/tf/appo_tf_learner.py

```diff
@@ -72,7 +72,7 @@ def compute_loss_for_module(
             trajectory_len=rollout_frag_or_episode_len,
             recurrent_seq_len=recurrent_seq_len,
         )
-        if self.config.enable_env_runner_and_connector_v2:
+        if config.enable_env_runner_and_connector_v2:
```

Comment:
I wonder, if the new env runners work on APPO/IMPALA. In my test case they do not in the MA case where a list of episodes is tried to be `compressed_if_needed`

Reply:
IMPALA and APPO are WIP on the new EnvRunners, officially not supported yet.

https://docs.ray.io/en/master/rllib/rllib-new-api-stack.html